test/tests/parser/parse_vector_realvectorvalue/tests

```diff
@@ -0,0 +1,19 @@
+[Tests]
+  design = 'Parser.md'
+  issues = '#24337'
+
+  [parse_vector_value]
```

Comment:
These requirements still need to be cleaned up a little

It's not really vectors of vectors of values, that's handled by std::vector<std::vector<Real>>

The second exception is too technical

Reply:
Okay, I modified them.